docs/iot-platform-sdk/running-the-demo.md

```diff
@@ -454,63 +454,124 @@ The above directory contains several subdirectories, each representing a device,
+      "valueLen": -1,
+      "valueId": "package1_name",
+      "enc": "base64"
+    },
+    {
+      "module": "sdo_sys",
+      "msg": "write",
+      "valueLen": -1,
+      "valueId": "script_one.sh",
+      "enc": "base64"
+    },
+    {
+      "module": "sdo_sys",
+      "msg": "exec",
+      "valueLen": -1,
+      "valueId": "binsh-script1",
+      "enc": "base64"
+    },
+    {
+      "module": "sdo_sys",
+      "msg": "filedesc",
+      "valueLen": -1,
+      "valueId": "package2_name",
+      "enc": "base64"
+    },
+    {
+      "module": "sdo_sys",
+      "msg": "write",
+      "valueLen": -1,
+      "valueId": "script_two.sh",
+      "enc": "base64"
+    },
+    {
+      "module": "sdo_sys",
+      "msg": "exec",
+      "valueLen": -1,
+      "valueId": "binsh-script2",
+      "enc": "base64"
+    }
+]
+```
 !!! Note
-     When running the services as Docker*, only <sdo-iot-platform-sdk-root\>/demo/ocs/config/db and <sdo-iot-platform-sdk-root\>/demo/to0scheduler/config/redirect.properties, are configurable.
+     Using a single `exec` block you cannot execute multiple scripts on the Device.
+
+To avoid different configurations for different devices, a unified structure is followed for svi.json file which is described above and works for all devices.
 
-Given an OwnershipVoucher file with a unique deviceId/GUID, the following needs to be done to add it as a device in the directory <sdo-iot-platform-sdk-root\>/demo/ocs/config/db/v1/devices:
+!!! Note
+     When running the services as Docker*, only `<sdo-iot-platform-sdk-root\>/demo/ocs/config/db` and `<sdo-iot-platform-sdk-root\>/demo/to0scheduler/config/redirect.properties`, are configurable.
+
+Given an OwnershipVoucher file with a unique deviceId/GUID, the following needs to be done to add it as a device in the directory `<sdo-iot-platform-sdk-root\>/demo/ocs/config/db/v1/devices`:
 
 1. Create a folder named deviceId/GUID.
 
 2. Rename the Owner voucher file to 'voucher.json'. Move this file to the newly created directory.
 
 3. Create the psi.json and svi.json files under the same directory. A sample of both files are present at the sample device directory '1fae14fb-deca-405a-abdd-b25391b9d932'.
 
-The following is a sample Python\* script that, given an Owner voucher file as an argument, extracts the GUID and creates the previously mentioned device directory structure.
-
-This sample script operates under the assumption that both the script and voucher files are placed at \<sdo-iot-platform-sdk-root>/demo/ocs/config/db/v1/devices, and that this is the current working directory:
+The following is a sample Python\* script that, given an Owner voucher file as an argument, extracts the GUID and creates the previously mentioned device directory structure. This sample script operates under the assumption that both the script and voucher files are placed at `<sdo-iot-platform-sdk-root>/demo/ocs/config/db/v1/devices`, and that this is the current working directory:
```

Comment:
is it Ownership Voucher in place of Owner Voucher?

Reply:
Ownership Voucher